gradle-plugin/src/main/groovy/ch/yvu/teststore/plugin/TeststoreClient.groovy

```diff
@@ -58,4 +59,11 @@ class TeststoreClient {
 
         httpClient.postJson("/runs/$runId/results", jsonBuilder.toString())
     }
+
+    def collectTags() {
+        def device = InetAddress.getLocalHost().getHostName()
```

Comment:
InetAddress.getLocalHost() might throw an UnknownHostException which should be handled here (e.g. just take a default string like "unknown").